app/components/header/header.component.html

```diff
@@ -107,6 +107,18 @@
             <a href="#"><img src="./assets/stash.png" height="40" width="48" class="add-repository-button" title="Stash"></a>
             <span class="toolbar-Text">Stash</span>
         </li>
+
+        <!--Add Upstream Remote Respository-->
+        <li class="toolbar-Icons" onclick="addRemoteModal()">
+            <a href="#"><img src="./assets/addremote.png" height="40" width="48" class="add-repository-button" title="Add Remote"></a>
+            <span class="toolbar-Text">Remote</span>
+        </li>
+
+        <!--Sync-->
```

Comment:
I would remove Sync until the impl is complete.

Reply:
@colinwerner I have removed sync and also renamed addRemoteRepo() to setUpstreamRepo()